$OpenBSD: patch-common_atoms_list,v 1.1 2015/09/07 10:30:01 dcoppa Exp $

commit db087fdda72af25fcfbdf0ca34e349a6e6b3c7fa
Author: Uli Schlachter <psychon@znc.in>
Date:   Sun Jul 26 17:03:10 2015 +0200

Keep client order across restarts

This saves the order of clients in a property called AWESOME_CLIENT_ORDER on the
root window during shutdown. During startup, after managing all existing
windows, we force the client list into the order described by this property
(overwriting any changes that Lua possibly did).

This code should safely handle cases where the property doesn't contain all
existing clients or contains a client which doesn't exist anymore.

Signed-off-by: Uli Schlachter <psychon@znc.in>

--- common/atoms.list.orig	Sat Jan 10 20:36:35 2015
+++ common/atoms.list	Mon Sep  7 11:25:16 2015
@@ -61,3 +61,4 @@ WM_WINDOW_ROLE
 WM_CLIENT_LEADER
 XSEL_DATA
 WM_TAKE_FOCUS
+AWESOME_CLIENT_ORDER
